src/index.html

```diff
@@ -1,13 +1,71 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style/index.css">
+  </head>
+  <body class="body">
+    <div class="container">
+      <div class="card" data-qa="card">
+        <div class="card__picture">
+          <img
+            src="images/imac.jpeg"
+            alt="Buy it!"
+            class="card__img"
+          >
+        </div>
+
+        <div class="card__product-name">
+          <h1 class="card__h1">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h1>
+        </div>
+
+        <div class="card__product-id">
+          <p class="card__p">Код товара: 195434</p>
+        </div>
+
+        <div class="card__reiting">
+          <div class="stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+            </div>
+          </div>
+
+          <div class="card__comments">
+            <p class="card__p">Отзывов: 5</p>
+          </div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-txt">
+            <p class="card__p">Цена:</p>
+          </div>
```

Comment:
I suggest that you can delete your wrappers for texts (you have just one element inside)

```suggestion
          <p class="card__price-hint">Цена:</p>
```